src/Workspaces/Core/Portable/Remote/RemoteArguments.cs

```diff
@@ -12,6 +12,24 @@ namespace Microsoft.CodeAnalysis.Remote
 {
     #region FindReferences
 
+    internal class SerializableFindReferencesSearchOptions
```

Comment:
😕 Is there no more direct way to make `FindReferencesSearchOptions` work with the serializer?

Reply:
Not that i'm aware of.

Reply:
Newtonsoft.JSON is the serializer and it can de/serialize `FindReferencesSearchOptions` directly just fine.

Reply:
Unfortunately, that's not hte case.  Or at least, it wasn't a version of VS ago.  The version of json.net being used could not handle these types.

Reply:
@CyrusNajmabadi Well, the remote tests passed for me without a Serializable type. If it would fail in VS, we should come up with a way to test that.